app-frontend/src/store/sagas/worker.js

```diff
@@ -10,10 +10,7 @@ export function* loadWorkersData(action) {
     const { payload } = action;
 
     const responseData = yield call(() => axios.get(
-      `${API_URL}/workers?pageNumber=${payload.page}
-			&positionType=${payload.filterParameters.positionType}
-			&sortingType=${payload.filterParameters.sortingType}
-			&time=${payload.filterParameters.time}`,
+      `${API_URL}/workers?pageNumber=${payload.page}&positionType=${payload.filterParameters.positionType}&sortingType=${payload.filterParameters.sortingType}&time=${payload.filterParameters.time}`,
```

Comment:
here is the way how to fix issue with long string:
https://github.com/axios/axios#browser

just build query params in the appropriate way.